amd_openvx_extensions/amd_rpp/source/copy.cpp

```diff
@@ -25,7 +25,7 @@ THE SOFTWARE.
 struct CopyLocalData
 {
 
-    RPPCommonHandle handle;
+    RPPCommonHandle *handle;
```

Comment:
We can remove RPPCommonHandle if its not used. 
Same for nop node.

Reply:
Its needed for the single handle creation